typed/reconcile_schema_test.go

```diff
@@ -98,6 +98,66 @@ func granularSchema(version string) typed.YAMLObject {
     - name: numeric
       type:
         scalar: numeric
+- name: empty
+  map:
+    elementType:
+      scalar: untyped
+      list:
+        elementType:
+          namedType: __untyped_atomic_
+        elementRelationship: atomic
+      map:
+        elementType:
+          namedType: __untyped_deduced_
```

Comment:
Does this already exist in our real schemas?

Reply:
Yes, here's a generated SMD schema (and it's corresponding openAPI schema) to show exactly type of schemas we're expecting: https://github.com/kubernetes/kube-openapi/pull/239/files#diff-6ba5d14994ba13458a380d3cf2e3b55bc5e15a823e34dc648bbe5ae442679254R52